platform/fabric/core/generic/ordering/ordering.go

```diff
@@ -133,6 +137,16 @@ func (o *Service) SetConsensusType(consensusType ConsensusType) error {
 	return nil
 }
 
+func (f *Service) SetConfigOrderers(o channelconfig.Orderer, orderers []*grpc.ConnectionConfig) error {
```

Comment:
We pass `o` that already contains `Consenters` and `orderers`. Do we need both? We could only pass ordererType.

Reply:
good point, we can simplify this.